test/letters/components/Address.unit.spec.jsx

```diff
@@ -58,5 +64,82 @@ describe('<Address>', () => {
     form.fillData('input[name="address"]', addressLine1);
     expect(onInputSpy.calledWith('addressOne', addressLine1)).to.be.true;
   });
-  // TODO: Add a test for rendering error messages
+
+  it('should populate country dropdown with countries prop', () => {
+    const component = ReactTestUtils.renderIntoDocument(<Address {...defaultProps}/>);
+    const form = getFormDOM(component);
+
+    const countriesElement = form.getElement('[name="country"]');
+    const countryNames = Array.from(countriesElement.options).map(o => o.value);
+
+    // The dropdown has an option for blank; clear that out before comparing
+    countryNames.shift();
+    expect(countryNames).to.eql(defaultProps.countries);
+  });
+
+  it('should add military state codes to the state dropdown', () => {
+    const component = ReactTestUtils.renderIntoDocument(<Address {...defaultProps}/>);
+    const form = getFormDOM(component);
+
+    const statesElement = form.getElement('[name="state"]');
+    const stateNames = Array.from(statesElement.options).map(o => o.value);
+
+    // Check to make sure the stateNames contains all the military state codes
+    Array.from(MILITARY_STATES).forEach((code) => {
```

Comment:
I was struggling to figure out how to test this earlier, so I learned something from reviewing this!